Makefile

```diff
@@ -48,11 +48,15 @@ mattermost_$(version)-$(revision).deb:
 	mkdir -p ${build_path}/${install_path}
 	tar xf ${tarname}.tar.gz --strip-components=1 -C ${build_path}/${install_path}
 
+	@echo "Renaming config file"
+	mv ${build_path}/opt/mattermost/config/config.json \
+	   ${build_path}/opt/mattermost/config/config.defaults.json
+
 	@echo "Creating binary links"
-	mkdir -p ${build_path}/usr/local/bin
```

Comment:
won't this change affect omnibus? I dunno that's why I am asking

Reply:
Nope. Omnibus is tested and works as expected. 
mattermost binary symbolic link is still available on the OS